### PR TITLE
Remind to apply mappings

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Tackles [comment](https://github.com/EGI-Federation/documentation/commit/619999870000c8dc87bf2dd71c2ea52d7b82228c#commitcomment-80596800). Adds the command to apply the 
mappings once the file has been updates

---

**Related issue :**

### CODE REVIEW

**Comment by gwarf** [comment]
I'm fixing some of the linting errors on other files in https://github.com/EGI-Federation/documentation/pull/485.
Once #485 will be merged we will rebase and see how it looks.

**Comment by gwarf** [comment]
The check-link error is a known false positive due to how the https://docs.egi.eu/providers/cloud-compute/openstack/ section is organised (with files directly under the main folder).

File: content/en/providers/cloud-compute/openstack/aai.md [hunk 1]
```diff
@@ -776,6 +776,11 @@ the URLs on your configuration:
      ]
    }
    ```
+1. Apply the updated mappings to your Keystone configuration:
```
Comment (gwarf) [inline content/en/providers/cloud-compute/openstack/aai.md:779]:
```suggestion

   ```
1. Apply the updated mappings to your Keystone configuration:
```

**Review by gwarf — APPROVED** [review]
LGTM

**Comment by enolfc** [comment]
I'm merging despite the (known) issue with the link checker

**Pull request merged**